Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -235,6 +235,14 @@ def network_request_will_be_sent(self, documentURL, **kwargs):
         '''Triggered when a request is sent by the browser, catches mailto URLs.'''
         demisto.debug(f'PychromeEventHandler.network_request_will_be_sent, {documentURL=}')
         self.is_mailto = documentURL.lower().startswith('mailto:')
+
+        request_url = kwargs.get('request', {}).get('url', '')
+        request_id = kwargs.get('requestId', '')
+        if "cloudflare" in request_url:
+            self.tab.Fetch.enable()
```

Comment:
Can't `enable` be called once, when starting the tab?

Need to `disable` afterwards?